packages/votura-validators/src/objects/election.ts

```diff
@@ -96,14 +96,19 @@ export const selectableElectionObjectSchema = z.toJSONSchema(
   toJsonSchemaParams,
 );
 
-export const updateableElectionObject = electionObject.pick({
-  name: true,
-  description: true,
-  votingStartAt: true,
-  votingEndAt: true,
-  allowInvalidVotes: true,
-  private: true,
-});
+export const updateableElectionObject = electionObject
+  .pick({
+    name: true,
+    description: true,
+    votingStartAt: true,
+    votingEndAt: true,
+    allowInvalidVotes: true,
+    private: true,
+  })
+  .refine((data) => data.votingStartAt < data.votingEndAt, {
```

Comment:
Please create a standalone function for this so we don't need to refactor in multiple places.